src/functions.date.js

```diff
@@ -108,28 +108,43 @@ for (let unit in s) {
 	}, {multiValued: true});
 }
 
-_.duration = $.extend(function($this, ms) {
+_.duration = $.extend(function($this, ms, terms = 1) {
 	if (arguments.length === 1) {
 		[ms, $this] = [$this, null];
```

Comment:
Also, another thing I forgot in my previous review: You added a parameter, but did not change this part of the code (which essentially deals with `$this` not being provided).

Reply:
I am a bit confused as to how we know whether $this has been provided if we have 2 arguments. Would it be that the type of $this is a number when it should be an object?

Reply:
It's not that simple, because `ms` can also come from a property, in which case they would be indistinguishable. Let me think about it and get back to you.